src/app/api/v1/lgtm-images/_utils/index.ts

```diff
@@ -0,0 +1,70 @@
+import getErrorData from "@/lgtm-data/error";
+import { NextRequest } from "next/server";
+import { ERRORS } from "../_constants";
+import { ImageResponse } from "next/og";
+import { GetLgtmData, InputData } from "@/types/lgtm-data";
+
+export function getSearchParams(request: NextRequest) {
+  const searchParams = request.nextUrl.searchParams;
+  return {
+    theme: searchParams.get("theme"),
+    text: searchParams.get("text"),
+    emoji: searchParams.get("emoji"),
+    color: searchParams.get("color"),
+  };
+}
+
+export function handleMissingTheme(inputData: InputData) {
+  const { element, options } = getErrorData({
+    ...inputData,
+    text: ERRORS.THEME_MISSING,
+  });
+  return new ImageResponse(element, { ...options, status: 400 });
```

Comment:
```suggestion
  return createErrorResponse(inputData, ERRORS.THEME_MISSING, 400);
```

他のエラーハンドリングと合わせて、`createErrorResponse()`を使用した方が良いかも？

Reply:
こちらもおっしゃる通りですね！！

`createErrorResponse ()`で、共通化しました👍